Formula/moar.rb

```diff
@@ -0,0 +1,24 @@
+class Moar < Formula
+  desc "Nice to use pager for humans"
+  homepage "https://github.com/walles/moar"
+  url "https://github.com/walles/moar.git",
+    using:    :git,
+    tag:      "v1.8.1",
+    revision: "fec9ad814970ee1a271b19326214647460b3200c"
+  license "BSD-2-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    system "./build.sh"
+    bin.install "moar"
```

Comment:
Let's build this the same way we do for other `go`-based formulae, take a look at `driftctl` for an example.